specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/privateEndpoint.json

```diff
@@ -306,7 +306,7 @@
     },
     "/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/{location}/availablePrivateEndpointTypes": {
       "get": {
-        "operationId": "AvailablePrivateEndpointTypes_List",
+        "operationId": "PrivateEndpoints_ListAvailablePrivateEndpointTypes",
```

Comment:
This kind of naming seems to trigger new warnings like this:
```
Per the Noun_Verb convention for Operation Ids, the noun 'PrivateEndpoints' should not appear after the underscore. Note: If you have already shipped an SDK on top of this spec, fixing this warning may introduce a breaking change.
```
Could you please consider naming operations w/o noun on the right side of underscore? E.g. `PrivateEndpoints_ListAvailableTypes` or something like this.

Reply:
Hi Evgeny, the AvailablePrivateEndpointType is a single noun. It's a single class type, different from privateEndpoint. So let's consider to still use 'PrivateEndpoints_ListAvailablePrivateEndpointTypes'. Thank you.

Reply:
I haven't noticed this earlier, but `AvailablePrivateEndpointTypes.List` has already been published in Network SDK and is being used in Azure PowerShell. So changing it would be a **breaking change** and we wouldn't want that. Could you please revert this change?
Also, I think in this case we would also need to change  `PrivateEndpoints_ListPrivateEndpointAvailableTypesByResourceGroup` to `AvailablePrivateEndpointTypes_ListByResourceGroup` to keep consistent naming with `List`/`ListByResourceGroup` pair.

Reply:
@number213 Thanks , that was my concern too. I have already revert it back and add _ListByResourceGroup.